v2/yarpcgrpc/integration_test.go

```diff
@@ -405,6 +406,11 @@ func TestTLS(t *testing.T) {
 	}
 }
 
+func wrapProcedures(procedures []yarpc.EncodingProcedure) []yarpc.TransportProcedure {
```

Comment:
This feels a little messy - very open to suggestions for other ways of doing this

Reply:
We can do better here! Let's change `testEnvOptions` to take in `EncodingProcedure`s and then `newTestEnv()` can convert them to `TransportProcedure`, with error handling.